frontend/src/components/Home/Autocomplete.tsx

```diff
@@ -58,12 +59,17 @@ export default function Autocomplete() {
   const [query, setQuery] = useState('');
   const [selected, setSelected] = useState<LandlordOrApartmentWithLabel | null>(null);
   const [width, setWidth] = useState(inputRef.current.offsetWidth);
+  const history = useHistory();
 
   function handleListKeyDown(event: React.KeyboardEvent) {
     event.preventDefault();
     if (event.key === 'Tab') {
       setOpen(false);
     }
+    if (event.key === 'Enter') {
```

Comment:
This function `handleListKeyDown` is only for the `Menu` component so let's not make any changes to it